.gitignore

```diff
@@ -121,3 +121,7 @@ setup.log
 
 # Merlin configuring file for Vim and Emacs
 .merlin
+
+.vscode/
```

Comment:
FYI I have this line in my global `~/.gitignore` so I don't have to add it to each repository where I use VS Code. But I don't mean to say that it shouldn't be added here.

Reply:
smart.

Reply:
I'm fine with adding it here, fwiw. But yeah

Reply:
Eh, I added a global .gitignore, so we can get rid of those lines. According to the internet, it's more kosher not to have them in a repo's .gitignore